journal-of-medical-internet-research.csl

```diff
@@ -218,6 +238,16 @@
       </if>
     </choose>
   </macro>
+  <macro name="webcite">
+    <choose>
+      <if type="webpage post-weblog" match="any">
+        <group delimiter=": " prefix=". ">
+          <text value="archived at" text-case="capitalize-first"/>
+          <text variable="note"/>
```

Comment:
I'm skeptical about this. This is one journal; we can't really expect users to change their data entry patterns for this and on the negative side this is going to add all sorts of incomprehensible clutter to the bibliography. I'd say comment this out